packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -41,386 +41,415 @@
 
 <body>
 
-<div class="examples-page">  
-  <div class="sidebar" id="sidenav"></div>
-  <div id="toggle"></div>
+  <div class="examples-page">
```

Comment:
I forgot to mention; we need to turn off HTML formatting in VSCode because it'll mess up how the examples look. I think you can just revert this since I don't think we're planning to add anything here, right?